### PR TITLE
WIP: Story Cat

### PR DESCRIPTION
**Pull request by chadbailey59**
Story Cat? Storycat? StoryCat? Whichever one, it's a cat that tells stories.

### CODE REVIEW

File: src/examples/starter-apps/chatbot.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        sprites.append(img.tobytes())
+
+flipped = sprites[::-1]
+sprites.extend(flipped)
+# When the bot isn't talking, show a static image of the cat listening
+quiet_frame = ImageFrame("", sprites[0])
+talking_frame = SpriteFrame(images=sprites)
+
+
+class TalkingAnimation(AIService):
```
Comment (Moishe) [inline src/examples/starter-apps/chatbot.py:55]: Should this have FrameProcessor as a parent?

File: src/examples/starter-apps/storycat.py [hunk 2]
```diff
@@ -57,278 +59,105 @@
...
+class StoryPromptFrame(TextFrame):
+    pass
 
-    def verify_birthday(self, args):
-        return args["birthday"] == "1983-01-01"
 
-    def list_prescriptions(self, args):
-        # print(f"--- Prescriptions: {args['prescriptions']}\n")
-        pass
+class StoryProcessor(AIService):
```
Comment (Moishe) [inline src/examples/starter-apps/storycat.py:74]: Should this descend from FrameProcessor rather than AIService?

**Review by Moishe — COMMENTED** [review]
Couple random comments but this is looking cool!

File: src/examples/starter-apps/chatbot.py (hunk 1 above)
Comment (chadbailey59) [reply src/examples/starter-apps/chatbot.py:56]: This one is still using `run_to_queue`, so it doesn't like being a `FrameProcessor` :(

**Comment by chadbailey59** [comment]
Closed because of all the formatting conflicts. See newer PR

**Pull request closed without merging**